### ISSUE TITLE
alter Table Command

### ISSUE THREAD

**Issue by telion2** [interactionlab/NLATool#28]
I want to automatically change the Settings of the content of the Database. To do this there is the alter Table
Command in MySql. To implement this Command (That means to create the String that contains the final Command) is not trivial, depending on the needs. 
Since I want to get moving forward I suggest to only implement parts of this command, but I dont know which parts I should prioritise except those I directly need. 

Here is what this Command is able to do:
https://dev.mysql.com/doc/refman/5.7/en/alter-table.html

My Plan is to implement all parts I need to change 
1. Database name (with ALTER DATABASE) (low priority)
2. Table names (high priority)
3. Column Names and Settings (high priority) (By settings I mean all that are specified in the dbconfig.json)

For Clarification: The challenge of implementing this Command is implementing it in such a way that you can make one MySql Request that changes everything on the DB. This requires a good way to input various types of data into that command. 
For e.g. the command ALTER TABLE word ADD COLUMN newColumnName VARCHAR (20) NOT NULL, DROP COLUMN wordID, ADD UNIQUE KEY semanticClass; 

The parameters for that are: tablename, newColumnName, type, Length, null/not null, operation (add, drop,..), addressed Columns, Columns that get their Key changed and so on.  

My Plan is to store all this in an Object but there will be things that need a specific syntax and so on. 

So to sum it up: 
I want to know if there aspects of ALTER TABLE that I need to focus on too. You can see which operations I probably implement in alterDB in DB-Actions line 26.
